### PR TITLE
Added option to skip kind in completion

### PR DESCRIPTION
**Pull request by yyoncho**

### CODE REVIEW

File: lsp-mode.el [hunk 1]
```diff
@@ -800,6 +800,12 @@ ignored."
   :group 'lsp-mode
   :package-version '(lsp-mode . "6.3.2"))
 
+(defcustom lsp-completion-show-kind t
+  "Whether or not to show kind of completion candidates."
+  :type 'boolean
+  :group 'lsp-mode
+  :package-version '(lsp-mode . "7.0.1"))
```
Comment (ericdallo) [inline lsp-mode.el:807]: As this is a new feature, shouldn't this be `7.1.0`?

Comment (ericdallo) [reply lsp-mode.el:807]: Oh, forget it, it's a existent feature

Comment (yyoncho) [reply lsp-mode.el:807]: @kiennq started using 7.0.1. Both with 7.1.0 work for me. We may add snapshot in the end, e. g. `7.1.0-SNAPSHOT`.

Comment (ericdallo) [reply lsp-mode.el:807]: I like to follow semantic versioning on `lsp-dart`, with `x.1.x` being new features and `x.x.1` being a patch/fix

Comment (yyoncho) [reply lsp-mode.el:807]: > I like to follow semantic versioning on `lsp-dart`, with `x.1.x` being new features and `x.x.1` being a patch/fix

I am fine with this approach.

**Pull request merged**